src/identity/components/userprofile/DefaultDropdown.tsx

```diff
@@ -0,0 +1,62 @@
+// Libraries
```

Comment:
This is a wrapper component for the change-default-account and change-default-org dropdowns.